easybuild/easyconfigs/p/PostgreSQL/PostgreSQL-15.1-GCCcore-12.2.0.eb

```
@@ -0,0 +1,43 @@
+easyblock = 'ConfigureMake'
+
+name = 'PostgreSQL'
+version = '15.1'
+
+homepage = 'https://www.postgresql.org/'
+description = """PostgreSQL is a powerful, open source object-relational database system.
+ It is fully ACID compliant, has full support for foreign keys,
+ joins, views, triggers, and stored procedures (in multiple languages).
+ It includes most SQL:2008 data types, including INTEGER,
+ NUMERIC, BOOLEAN, CHAR, VARCHAR, DATE, INTERVAL, and TIMESTAMP.
+ It also supports storage of binary large objects, including pictures,
+ sounds, or video. It has native programming interfaces for C/C++, Java,
+ .Net, Perl, Python, Ruby, Tcl, ODBC, among others, and exceptional documentation."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
+
+source_urls = ['http://ftp.postgresql.org/pub/source/v%(version)s/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['ea2cf059a85882654b989acd07edc121833164a30340faee0d3615cf7058e66c']
+
+builddependencies = [
+    ('binutils', '2.39'),
+    ('Bison', '3.8.2'),
+    ('flex', '2.6.4'),
+    ('Perl', '5.36.0'),
+    ('Python', '3.10.8'),
+]
+
+dependencies = [
+    ('libreadline', '8.2'),
+    ('zlib', '1.2.12'),
+#   ('OpenSSL', '1.1', '', SYSTEM),
```

@SimonPinches I see you're commenting out `OpenSSl` a lot
if you're wanting to rely on the OS-provided `openssl` instead of an easybuild provided one, you can just force-build the `OpenSSl` [wrapper easyconfig](https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/o/OpenSSL/OpenSSL-1.1.eb) (`OpenSSL-1.1.eb`) with `wrap_system_openssl = False`
```suggestion
    ('OpenSSL', '1.1', '', SYSTEM),
```